source/_components/homematicip_cloud.markdown

```diff
@@ -13,7 +13,7 @@ ha_release: 0.66
 featured: false
 ---
 
-The [HomematicIP](http://www.homematicip.com/) component platform is used as an interface to the cloud server.
+The [HomematicIP](http://www.homematicip.de) component platform is used as an interface to the cloud server.
```

Comment:
Incorrect. The current URL is indeed wrong, but your suggestion is incorrect as well.
It should be: `https://www.homematic-ip.com`